### PR TITLE
Dotnet 3.1.2 async

### PR DESCRIPTION
**Pull request by koenmetsu**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -8,6 +8,9 @@ wwwroot/
 .fake/
 .idea/
 .ionide/
+.paket/
+
+**/*/mono_crash.*
```
Comment (CumpsD) [inline .gitignore:13]: :laughing:

File: src/OrganisationRegistry.VlaanderenBeNotifier/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/dotnet/core/runtime-deps:2.2.4
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1.1
```
Comment (CumpsD) [inline src/OrganisationRegistry.VlaanderenBeNotifier/Dockerfile:1]: `3.1.2`

File: src/OrganisationRegistry.Api/Report/Responses/ClassificationOrganisationParticipation.cs [hunk 3]
```diff
@@ -120,17 +120,22 @@ public class ClassificationOrganisationParticipation
                         .Where(mandate => mandate.BodyId == bodyId)
                         .Where(mandate => activeSeatIds.Contains(mandate.BodySeatId))
                         .Where(mandate =>
-                            (!mandate.BodyMandateValidFrom.HasValue || mandate.BodyMandateValidFrom <= DateTime.Today) &&
-                            (!mandate.BodyMandateValidTo.HasValue || mandate.BodyMandateValidTo >= DateTime.Today));
+                            (!mandate.BodyMandateValidFrom.HasValue ||
+                             mandate.BodyMandateValidFrom <= DateTime.Today) &&
```
Comment (CumpsD) [inline src/OrganisationRegistry.Api/Report/Responses/ClassificationOrganisationParticipation.cs:124]: Formatting of this `||` as before

File: src/OrganisationRegistry.Api/appsettings.json [hunk 4]
```diff
@@ -17,8 +17,8 @@
   ],
 
   "SqlServer": {
-    "ConnectionString": "Server=localhost,21433;Database=OrganisationRegistry;User ID=sa;Password=E@syP@ssw0rd;",
-    "MigrationsConnectionString": "Server=localhost,21433;Database=OrganisationRegistry;User ID=sa;Password=E@syP@ssw0rd;"
+    "ConnectionString": "Server=localhost,21433;Database=OrganisationRegistry;User ID=sa;Password=E@syP@ssw0rd;MultipleActiveResultSets=True",
```
[the remark this answers is not in the document]
Comment (koenmetsu) [reply src/OrganisationRegistry.Api/appsettings.json:20]: yes

File: src/OrganisationRegistry.Api/Body/Queries/BodyBodyClassificationListQuery.cs [hunk 5]
```diff
@@ -4,6 +4,7 @@ namespace OrganisationRegistry.Api.Body.Queries
     using System.Collections.Generic;
     using System.Linq;
     using System.Linq.Expressions;
+    using Be.Vlaanderen.Basisregisters.Api.Search.Helpers;
```
[the remark this answers is not in the document]
Comment (koenmetsu) [reply src/OrganisationRegistry.Api/Body/Queries/BodyBodyClassificationListQuery.cs:7]: nope, good catch.

File: src/OrganisationRegistry.Api/appsettings.json (hunk 4 above)
[the remark this answers is not in the document]
Comment (koenmetsu) [reply src/OrganisationRegistry.Api/appsettings.json:20]: more specifically, if we go async, we need this during reactions.

File: src/OrganisationRegistry.Api/Report/Responses/BodyParticipation.cs [hunk 6]
```diff
@@ -98,7 +98,8 @@ public class BodyParticipation
                     .Where(mandate => activeSeatIds.Contains(mandate.BodySeatId))
                     .Where(mandate =>
                         (!mandate.BodyMandateValidFrom.HasValue || mandate.BodyMandateValidFrom <= DateTime.Today) &&
-                        (!mandate.BodyMandateValidTo.HasValue || mandate.BodyMandateValidTo >= DateTime.Today));
+                        (!mandate.BodyMandateValidTo.HasValue || mandate.BodyMandateValidTo >= DateTime.Today))
+                    .ToList();
```
[the remark this answers is not in the document]
Comment (koenmetsu) [reply src/OrganisationRegistry.Api/Report/Responses/BodyParticipation.cs:102]: We can, but I'm gonna leave it there just for clarity.

**Pull request merged**